matrix/common/dynamo_utils.py

```diff
@@ -115,9 +115,9 @@ def acquire(self):
             # Or maybe we hold the lock ourselves, then just return
             elif db_response["Item"]["LockHolder"] == self._lock_id:
                 return
-
+            print(f"Waiting for lock on {self._lock_key}")
```

Comment:
Does it make sense to use logger here?

Reply:
Oh yeah, sorry this was something I dropped in and forgot about.